app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -3946,16 +3946,26 @@ class BrowserTabViewModel @Inject constructor(
         newTabPixels.get().fireNewTabDisplayed()
     }
 
+    private fun fireReloadPixelIfRefreshCountMet() {
+        if (brokenSitePrompt.getUserRefreshesCount(reset = false) >= REFRESH_COUNT_LIMIT) {
+            Timber.d("KateTest--> Sending 3x reload pixel from BTVM fireReloadPixelIfRefreshCountMet")
+            pixel.fire(AppPixelName.RELOAD_THREE_TIMES_WITHIN_20_SECONDS)
+        }
+    }
+
     fun handleMenuRefreshAction() {
         refreshPixelSender.sendMenuRefreshPixels()
+        fireReloadPixelIfRefreshCountMet()
```

Comment:
Can we move this logic to the RefreshPixelSender? We already have that class to do this so it seems like we are duplicating some logic. For example, the 3x refresh should also send the experimental pixels for the blocklist but it's not.